web/core/components/issues/parent-issues-list-modal.tsx

```diff
@@ -121,6 +121,7 @@ export const ParentIssuesListModal: React.FC<Props> = ({
                       value={searchTerm}
                       onChange={(e) => setSearchTerm(e.target.value)}
                       displayValue={() => ""}
+                      tabIndex={isMobile ? -1 : 1}
```

Comment:
Can we move this logic also inside the helper?

Reply:
Resolved